src/foam/graphics/TreeGraph.js

```diff
@@ -36,303 +38,68 @@
      }
    ],
 
+   topics: [
+    'onSizeChange'
```

Comment:
Sorry maybe my previous comment about this wasn't too clear. I'm pretty sure you can remove this topic completely and subscribe to the treegraph's width$ instead.

Reply:
Replaced both subs by making a sub to width$ and the search feature doesn't seem to work anymore:
<img width="1680" alt="Screen Shot 2019-09-27 at 3 33 28 PM" src="https://user-images.githubusercontent.com/26717171/65797036-2cf43500-e13c-11e9-80cd-d21b4497287e.png">